domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressListener.kt

```diff
@@ -0,0 +1,12 @@
+package org.oppia.android.domain.exploration
+
+import org.oppia.android.app.model.ProfileId
+
+/** Listener for when an exploration is started or ended. */
+interface ExplorationProgressListener {
```

Comment:
It's worthwhile to document whether this listener may be accessed on multiple threads (so that implementers can take precautions, if needed).

Reply:
I updated this Kdoc

Reply:
This looks better now. It can be marked as resolved.

Reply:
This has now been addressed.